pysc2/pysc2/lib/pythonUtils.js

```diff
@@ -156,3 +156,30 @@ module.exports = {
   withPython,
   zip,
 }
+
```

Comment:
Take note of modules.export

It alone holds a reference to everything that is created and exported from this file. Your function definition should go above it, and then add your function to the exports object